static/app/routes.tsx

```diff
@@ -1048,22 +1027,38 @@ function buildRoutes() {
         path=":projectId/events/:eventId/"
         component={errorHandler(ProjectEventRedirect)}
       />
+      <Route
+        path=":projectId/getting-started/"
+        component={make(
+          () => import('sentry/views/projectInstall/platformOrIntegration')
+        )}
+      />
```

Comment:
since we are now using the context "gettingStartedWithProject", both "new" and "getting-started" routes have to contain the same parent